lib/Onyx.js

```diff
@@ -732,7 +734,7 @@ function initializeWithDefaultKeyStates() {
         .then((pairs) => {
             const asObject = _.object(pairs);
 
-            const merged = lodashMerge(asObject, defaultKeyStates);
+            const merged = lodashMergeWith(asObject, defaultKeyStates, customizerForMergeWith);
```

Comment:
voodoo! how does this know which argument is the customizer? 😄

Reply:
Magic! heh I assume it looks at the type of the last param and if it is a function it assumes it's the customizer.